code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -5624,6 +5624,42 @@ obj/item/weapon/reagent_containers/food/snacks/butteredtoast
 	..()
 	reagents.add_reagent(NUTRIMENT, 2)
 
+/obj/item/weapon/reagent_containers/food/snacks/shanghaifry
```

Comment:
Can you make these like the chips so they have an empty box when finished eating? It would seem weird to just eat the box whole.